addons/pos_self_order/models/pos_order.py

```diff
@@ -10,17 +10,21 @@
 class PosOrderLine(models.Model):
     _inherit = "pos.order.line"
 
-    # For the moment we need this to keep attributes consistency between the server and client_side.
-    selected_attributes = fields.Json(string="Selected Attributes")
-
     # FIXME: uuid already pass in pos and move note in pos_restaurant.
     def _export_for_ui(self, orderline):
         return {
-            'uuid': orderline.uuid,
-            'note': orderline.note,
+            "uuid": orderline.uuid,
+            "note": orderline.note,
```

Comment:
useless diff

Reply:
@vlst-odoo In python, variable names must be surrounded by ' and in javascript by ". 
In python only the text intended for the user must be surrounded by ".

Reply:
i didn't intentionally change it. That's just what the formatter does. I'll change it back anyways